src/ServiceManagement/Compute/Commands.ServiceManagement.Preview/Automation/RunCodeGeneration.ps1

```diff
@@ -198,6 +199,56 @@ function Get-NormalizedTypeName
     return $outputName;
 }
 
+function Is-SpecialType
+{
+    param(
+        [Parameter(Mandatory = $True)]
+        [string]$inputName
+    )
+
+    return ($inputName.StartsWith($client_model_namespace_prefix));
```

Comment:
```client_model_namespace_prefix``` shall be an input parameter, so that you can turn this function into a general one that can be used elsewhere as well.